lib/expand.js

```diff
@@ -577,8 +577,12 @@ class Expander {
       // Remove the previous type constraint since this one supercedes it
       constraints = constraints.filter(cst => cst !== previous);
     }
-    constraints.push(constraint);
 
+    // As a code constraint is a fixed value, we no longer need to retain value set constraints
+    const vsConstraints = (new models.ConstraintsFilter(previousConstraints)).withPath(constraint.path).valueSet.constraints;
+    vsConstraints.forEach(prev => constraints = constraints.filter(cst => cst !== prev))
```

Comment:
Hmm... This code and the previously existing code that it's modeled after seem very inefficient.  We don't need to address it now, but let's try to remember it for later.